lib/App/index.js

```diff
@@ -2029,6 +2029,11 @@ class App {
       dirs.push( '.homeycompose' );
       dirs.push( path.join('.homeycompose', 'flow') );
```

Comment:
is the flow directory useful by itself? don't you always need either `flow/triggers`, `flow/conditions` or `flow/actions` folders?

Reply:
Hm, I think that is right